api/google_api/routes/sheets.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const { SceGoogleApiHandler } = require('../util/SceGoogleApiHandler');
+const {
+  OK,
+  BAD_REQUEST
+} = require('../../util/constants').STATUS_CODES;
+
+router.post('/addToSpreadsheet', async (req, res) =>{
+  const scopes = ['https://sheets.google.com/'];
+  const pathToToken = __dirname + '/../../config/sheets.json';
+  const apiHandler = new SceGoogleApiHandler(scopes, pathToToken);
+  const sheetsID = req.body.sheetsID;
+  const row = {
```

Comment:
you can also do
```suggestion
  const row = { req.body }
```

and it will have all the fields you care about. read about [js object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)